public/stylesheets/style.css

```diff
@@ -372,3 +384,172 @@ input:checked + .slider:before {
 .slider.round:before {
   border-radius: 50%;
 }
+
+/*CSS for text styles ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
```

Comment:
I feel like this can be refactored a bit by: using meaningful class names e.g. `primary-text` instead of text-style-15. You can remove a lot of duplication by using the same class in the html code so you could remove all the repeated `#000000` code by using just one class called `black-text` and use that in the html.

Reply:
I have changed the names to be more suiting, there are a lot of styles so I tried to make them not very generic.